tests/integrationtest/t/expression/misc.test

```diff
@@ -581,3 +581,33 @@ set @@tidb_enable_vectorized_expression = false;
 select length(@p);
 set @@tidb_enable_vectorized_expression = true;
 select length(@p);
+
+# TestNotExistFunc
+connect (conn1, localhost, root,,);
+
+## current db is empty
+-- error 1046
+SELECT xxx(1);
+-- error 1046
+SELECT yyy();
+-- error 1305
+SELECT T.upper(1);
+
+## current db is not empty
+use test;
```

Comment:
```suggestion
```

Reply:
It's by design. See L586, we didn't call `use databse` before. It's for test different error msg.

Reply:
ok